config/src/main/java/com/typesafe/config/ConfigMemorySize.java

```diff
@@ -0,0 +1,49 @@
+/**
+ *   Copyright (C) 2015 Typesafe Inc. <http://typesafe.com>
+ */
+package com.typesafe.config;
+
+/**
+ * An immutable class representing an amount of memory.  Use
+ * static factory methods such as {@link
+ * ConfigMemorySize#ofBytes(long)} to create instances.
+ */
+public final class ConfigMemorySize {
```

Comment:
i'd prefer calling it just MemorySize instead of ConfigMemorySize. i know a lot of the other stuff is prefixed with config, but generally as an end use the only stuff I interact with is a COnfig or ConfigFactory or something where it actually makes sense to have config in the name. it'd probably be nicer not to have config here